from discord.ext import commands
import asyncio
import traceback
import discord

bot = commands.Bot(command_prefix=('prefix'))
token = os.environ['DISCORD_BOT_TOKEN']

@bot.event
async def on_command_error(ctx, error):
    orig_error = getattr(error, "original", error)
    error_msg = ''.join(traceback.TracebackException.from_exception(orig_error).format())
    await ctx.send(error_msg)

@bot.command()
@commands.check(is_owner)
async def コマンド名(ctx):
    await ctx.send('送りたいメッセージ')
